src/Schema/Events/Click.ts

```diff
@@ -522,6 +521,37 @@ export interface ClickedNavigationTab {
   subject: string
 }
 
+/**
+ *  User clicks in one of the price options on the offer page
+ *
+ *  This schema describes events sent to Segment from [[clickedOfferOption]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedOfferOption",
+ *    flow: Make Offer
+ *    context_page_owner_type: "orders-offer",
+ *    context_page_owner_id: "dd0cbbb5-300b-4c49-92a1-fed55b077fa9"
+ *    order_id: "407dd09f-4afd-4aad-a6cc-1d6704dc2b11"
+ *    offer: [20% below the list price, 15% below the list price, 10% below the list price, low-end of range, midpoint of the range, top-end of the range, different amount]
+ *    amount: 2000
+ *    currency: USD
```

Comment:
This is a string so you need: "USD"
You also need a comma at the end of the line.